tests/conftest.py

```diff
@@ -462,3 +572,61 @@ def assert_shown_files(
     assert (
         len(row_data) == expected_count
     ), f"Expected {expected_count} items, but got {len(row_data)}"
+
+
+# --- Test data download/extraction helpers for acceptance tests ---
+@pytest.fixture(scope="module")
+def otcamera19_extracted_dir(test_data_dir: Path) -> Path:
```

Comment:
The workflow file already has a step to download the test data. Why do we need this here?